src/modules/72-freeze-windows/components/__tests__/FreezeServiceFieldRenderer.test.tsx

```diff
@@ -0,0 +1,243 @@
+          <ServiceField
+            allServiceChecked={false}
+            setAllServicesChecked={jest.fn()}
+            index={0}
+            getString={jest.fn()}
+            formikProps={{ setFieldValue: jest.fn() }}
+          />
+        </TestWrapper>
+      </Formik>
+    )
+
+    const servicesFieldElement = queryByAttribute(
+      'data-tooltip-id',
+      document.body,
+      'form2_entity[0].Service'
+    ) as HTMLInputElement
+    expect(servicesFieldElement).toBeDefined()
+    const excludeServiceCheckboxElement = queryByAttribute(
+      'name',
+      document.body,
+      'entity[0].ExcludeServiceCheckbox'
+    ) as HTMLInputElement
+    const excludeServiceCheckboxElementParent = excludeServiceCheckboxElement.parentElement
+    expect(excludeServiceCheckboxElementParent).toHaveClass('bp3-disabled')
+  })
+  test('All Services Checked', async () => {
+    render(
+      <Formik initialValues={{}} formName="form2" onSubmit={jest.fn()}>
+        <TestWrapper
+          path="/account/:accountId/settings/freeze-windows/studio/window/:windowIdentifier/"
+          pathParams={{ accountId, windowIdentifier: '-1' }}
+        >
+          <ServiceField
+            allServiceChecked={true}
+            setAllServicesChecked={jest.fn()}
+            index={0}
+            getString={jest.fn()}
+            formikProps={{ setFieldValue: jest.fn() }}
+          />
+        </TestWrapper>
+      </Formik>
+    )
+
+    const excludeServiceCheckboxElement = queryByAttribute(
+      'name',
+      document.body,
+      'entity[0].ExcludeServiceCheckbox'
+    ) as HTMLInputElement
+    const excludeServiceCheckboxElementParent = excludeServiceCheckboxElement.parentElement as HTMLInputElement
+    expect(excludeServiceCheckboxElementParent).toBeDefined()
+    expect(excludeServiceCheckboxElementParent).not.toHaveClass('bp3-disabled')
+    fireEvent.click(excludeServiceCheckboxElementParent)
+    //data-tooltip-id="form2_entity[0].ExcludeService"
+    const excludeServicesFieldElement = queryByAttribute(
+      'data-tooltip-id',
+      document.body,
+      'form2_entity[0].ExcludeService'
+    ) as HTMLInputElement
+    expect(excludeServicesFieldElement).toBeDefined()
+  })
+  test('All Services Checked with Excluded Services', async () => {
+    render(
+      <Formik initialValues={initialValues} formName="form2" onSubmit={jest.fn()}>
+        <TestWrapper
+          path="/account/:accountId/settings/freeze-windows/studio/window/:windowIdentifier/"
+          pathParams={{ accountId, windowIdentifier: '-1' }}
+        >
+          <ServiceField
+            allServiceChecked={true}
+            setAllServicesChecked={jest.fn()}
+            index={0}
+            getString={jest.fn()}
+            formikProps={{
+              setFieldValue: jest.fn(),
+              values: initialValues
+            }}
+          />
+        </TestWrapper>
+      </Formik>
+    )
+    const excludedServicesScope = screen.getByText('ACCOUNT')
```

Comment:
Then use the above getByText here. In this was it is a bounded function.
```suggestion
expect(getByText('ACCOUNT')).toBeInTheDocument()

```